test/integration/imagechange_buildtrigger_test.go

```diff
@@ -465,6 +465,8 @@ func TestMultipleImageChangeBuildTriggers(t *testing.T) {
 	}
 	defer watch2.Stop()
```

Comment:
Hmm granted you're not adding this code now, but I am failing to understand why we need the two watchers created with the same arguments...
And yes, I read the portion below where `watch` and `watch2` gets used.